src/Microsoft.AspNet.Mvc.Core/Rendering/Internal/TemplateRenderer.cs

```diff
@@ -7,16 +7,19 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using Microsoft.AspNet.Http;
 using Microsoft.AspNet.Mvc.Core;
+using Microsoft.AspNet.Mvc.ModelBinding;
 using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Internal;
 
-namespace Microsoft.AspNet.Mvc.Rendering
+namespace Microsoft.AspNet.Mvc.Rendering.Internal
 {
-    internal class TemplateRenderer
+    public class TemplateRenderer
     {
         private static readonly string DisplayTemplateViewPath = "DisplayTemplates";
         private static readonly string EditorTemplateViewPath = "EditorTemplates";
+        public static readonly string IEnumerableOfIFormFileName = "IEnumerable`" + nameof(IFormFile);
```

Comment:
Nope didn't miss anything. Whenever there's a `public` class **I** always assume `static readonly`. For internal namespaced classes, could definitely choose not too. 

I like to go with the approach that's more flexible and leads to less head scratching/searching when re-reading code down the line (as long as its impact is not significant). And heck maybe it's just me that finds it easier/quicker to assume `public` => `public static readonly` instead of `const` but then again here we are in this long convo :smile: 

<- Indifferent for this situation.